addons/point_of_sale/models/pos_order.py

```diff
@@ -558,6 +558,18 @@ def refund(self):
             'target': 'current',
         }
 
+    def action_receipt_to_customer(self, receipt, cashier_id, client):
+        template_obj = self.env['mail.mail']
```

Comment:
you didn't do for loop because it works on only one record. so you need to do self.ensure_one()